CHANGELOG.md

```diff
@@ -29,7 +29,7 @@
 ### FEATURES:
 * **New builder:** `lxc` for building lxc images. [GH-3523]
 * **New builder:** `lxd` for building lxd images. [GH-3625]
-* **New builder**: Oracle Bare Metal Cloud Services (BMCS) builder for creating custom images. [GH-4554]
+* **New builder**: Oracle Cloud Infrastructure (OCI) builder for creating custom images. [GH-4554]
```

Comment:
in the future we try not to make changes to changelog in PR because of conflicts

Reply:
@mwhooker Happy to remove if preferable?

Reply:
yeah I think that would be best, thanks!